File: 01455.py
N, M = map(int, input().split())

coins = []
for i in range(N):
    s = input()
    coin = []
    for j in range(len(s)):
        coin.append(int(s[j]))

    coins.append(coin)

cnt = 0
for i in range(N - 1, -1, -1):
    for j in range(M - 1, -1, -1):
        if coins[i][j] == 1:
            cnt += 1
            for a in range(i + 1):
                for b in range(j + 1):
                    if coins[a][b] == 0: coins[a][b] = 1
                    else: coins[a][b] = 0

print(cnt)

File: 01455.cpp
#include <iostream>
using namespace std;

char coins[100][100];

int main() {
	int N, M; cin >> N >> M;
	for (int i = 0; i < N; i++)
		for (int j = 0; j < M; j++)
			cin >> coins[i][j];

	int cnt = 0;
	for (int i = N - 1; i >= 0; i--) {
		for (int j = M - 1; j >= 0; j--) {
			if (coins[i][j] == '1') {
				cnt++;
				for (int a = 0; a <= i; a++)
					for (int b = 0; b <= j; b++) {
						if (coins[a][b] == '0') coins[a][b] = '1';
						else coins[a][b] = '0';
					}
			}
		}
	}

	cout << cnt << endl;
}
